server/routes/command_policies.py

```
@@ -0,0 +1,370 @@
+    updates.append("updated_at = %s")
+    params.append(datetime.utcnow())
+    updates.append("updated_by = %s")
+    params.append(user_id)
+    params.extend([rule_id, org_id])
+
+    from utils.db.connection_pool import db_pool
+    with db_pool.get_admin_connection() as conn:
+        with conn.cursor() as cur:
+            cur.execute(
+                f"UPDATE org_command_policies SET {', '.join(updates)} "
+                "WHERE id = %s AND org_id = %s",
```

Ruff (0.15.10)

[error] Possible SQL injection vector through string-based query construction

(S608)